src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/domene/ÅrsakAvslagDvh.kt

```diff
@@ -11,6 +11,9 @@ enum class ÅrsakAvslagDvh {
     RETT_TIL_UTSTYRSSTIPEND,
     HAR_IKKE_MERUTGIFTER,
     RETT_TIL_BOSTØTTE,
+    REISEAVSTAND_UNDER_6_KM,
+    LØNN_I_TILTAK_ELLER_ORDINAR_LØNN,
```

Comment:
```suggestion
    LØNN_I_TILTAK_ELLER_ORDINÆR_LØNN,
```
Hvis vi bruker **Ø** kan vi vel bruke **Æ** også? 😁